app/commands/git/sync_concept.rb

```diff
@@ -23,7 +23,6 @@ def call
     attr_reader :concept
 
     def concept_needs_updating?
-      return false if synced_to_head?
```

Comment:
This was not needed as the synced_to_head check was already done at the level of the track syncing (which calls this file's class)